lib/elasticband/query.rb

```diff
@@ -136,7 +95,7 @@ def parse_boost_function(boost_options)
         if boost_options[:boost_by].present?
           ScoreFunction::FieldValueFactor.new(boost_options[:boost_by], modifier: :ln2p)
         elsif boost_options[:boost_where].present?
-          filter = join_filters(parse_filters(boost_options[:boost_where]))
+          filter = Filter.parse(only: boost_options[:boost_where])
           ScoreFunction::Filtered.new(filter, ScoreFunction::BoostFactor.new(1_000))
```

Comment:
No, it's just a big boost based on searchkick. I think we can do it configurable later